client/my-sites/plans/utils.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Internal dependencies
+ */
+
+import { abtest } from 'lib/abtest';
+import { getSavedVariations } from 'lib/abtest';
+
+function isInFreeTrialTest() {
+	if ( getSavedVariations().freeTrials_20160107 && abtest( 'freeTrials' ) === 'offered' ) {
```

Comment:
If we call `abtest` directly then the user will be assigned to the test if they aren't in it already. However if we find out the saved variation first then we can call `abtest` only for user who are already in a test.